modules/home-environment.nix

```diff
@@ -206,6 +206,12 @@ in
       '';
     };
 
+    home.sessionPath = mkOption {
+      type = types.envVar;
+      description = "Extra directories to add to the $PATH";
+      default = "";
```

Comment:
Can you give an example value with multiple paths?

Reply:
updated :+1: